Create Rule “upgrade-your-laptop/rule” (#4120)
SSWConsulting/SSW.Rules.Content: rules/upgrade-your-laptop/rule.md

---
type: rule
title: Do you know when to upgrade your laptop?
uri: upgrade-your-laptop
authors:
  - title: Ash Anil
    url: https://www.ssw.com.au/people/ash
created: 2022-09-27T05:52:37.565Z
guid: 52b3f2c6-fcb8-42f6-b469-f1621f839b49
---
Electronic gadgets should have an expiry date because their performance fades as we use them frequently.

> For example, laptops are the IT employee's best friend which helps to code and make life easier. We spend thousands of $ and get a laptop with high specs and think it would be good for the next 10 years or so. 

Reviewer: ```suggestion
For example, in the IT industry, laptops are the tools of the trade. Employees often spend thousands of $ and get a laptop with high specs thinking it will be good for the next 10 years or so.
```

Author: Done

> The problem with these laptops or any electronic devices (mobile) are that, as we use them the processor gets weaker and doesn't give the same performance as when it was new. 

Reviewer: ```suggestion
The problem with laptops and other electronic devices ( e.g. mobiles) is that, as we use them, the processor gets weaker and doesn't give the same performance as when it was new. 
```

> Most experts estimate a laptop’s lifespan to be three to five years. It may survive longer than that, but its utility will be limited as the components become less capable of running advanced applications. https://www.empowerit.com.au/blog/it-planning/budgeting/laptops-desktops-how-long-do-they-last/ 

Reviewer: ```suggestion
Most experts estimate a laptop’s lifespan to be 3 to 5 years. While a laptop might survive longer, its utility will be limited as the components become less capable of running advanced applications. https://www.empowerit.com.au/blog/it-planning/budgeting/laptops-desktops-how-long-do-they-last/ 
```

Also note that the source contradicts what you say about laptop lifespan, consider aligning the timelines, and adding the link inline using the syntax []()

> To keep track of the laptop's lifespan, we can use asset management tools like snipe https://snipeitapp.com/ and use the audit option to set a date after 5 years (recommended) to notify us to upgrade the laptop 

Reviewer: ```suggestion
To keep track of a laptop's lifespan, we can use asset management tools like snipe https://snipeitapp.com/ and use the audit option. That option allows a review date to be set after a period of time (5 years is recommended) so that the user knows to upgrade the laptop 
```

![❌Figure: Bad example - Old laptops pilled up in server room](microsoftteams-image-7-.png)

> These are all assets tagged in Snipe (asset management) and older than 5 years. 

Reviewer: ```suggestion
These are all assets tagged in Snipe that are older than 5 years. 
```

Don't need asset management, it was already mentioned

> We have a policy to get rid of laptops older than 5 years by selling them on eBay or if not working E-waste 

Reviewer: I would turn this into a different section under a heading like "What do do with an old laptop?", I would also add a heading higher up to say "How long do laptops generally last?"

---
type: rule
title: Do you know when to upgrade your laptop?
uri: upgrade-your-laptop
authors:
  - title: Ash Anil
    url: https://www.ssw.com.au/people/ash
created: 2022-09-27T05:52:37.565Z
guid: 52b3f2c6-fcb8-42f6-b469-f1621f839b49
---
Electronic gadgets should have an expiry date because their performance fades as we use them frequently.

For example, in the IT industry, laptops are the tools of the trade. Employees often spend thousands of $ and get a laptop with high specs and thinking it will be good for the next 10 years or so. 

The problem with laptops and other electronic devices (e.g. mobiles) is that, as we use them, the processor gets weaker and doesn't give the same performance as when it was new. 

Most experts estimate a laptop’s lifespan to be 3 to 5 years. While a laptop might survive longer, its utility will be limited as the components become less capable of running advanced applications. https://www.empowerit.com.au/blog/it-planning/budgeting/laptops-desktops-how-long-do-they-last/ 

> ### How long do laptops generally last?

Reviewer: not the right spot

---
type: rule
title: Do you know when to upgrade your laptop?
uri: upgrade-your-laptop
authors:
  - title: Ash Anil
    url: https://www.ssw.com.au/people/ash
created: 2022-09-27T05:52:37.565Z
guid: 52b3f2c6-fcb8-42f6-b469-f1621f839b49
---
Electronic gadgets should have an expiry date because their performance fades as we use them frequently. Having a track of lot of computers is not an easy task, even though we use an asset management tool.  

> For example, in the IT industry, laptops are the tools of the trade. Employees often spend thousands of $ and get a laptop with high specs and think it will be good for the next 10 years or so. We might not notice, but the old laptops don't give a good performance, which can reduce the production time of an employee causing a great loss for the company.

Reviewer: ```suggestion
For example, in the IT industry, laptops are the tools of the trade. Employees often spend thousands of $ and get a laptop with high specs and think it will be good for the next 10 years or so. It's easy to miss because we use them everyday, but old laptops usually don't have good performance, which can reduce the productivity an employee.
```

### Rotation - Keep the fleet up-to-date

The laptops in your fleet should be fit for purpose and performant. In the old days, people used to [Benchmark their laptops](https://www.ssw.com.au/rules/do-you-benchmark-your-pc) and it's still useful for personal machines, however, it is not the best method for enterprise asset management. 

Enterprises should buy laptops with standardized specs, in batches, at regular intervals e.g. every 6 months. Then the laptops should be rotated for employees on a regular cadence e.g. every 5 years. 

#### How do you figure out what cadence you replace laptops at?

> Most experts estimate a laptop’s lifespan to be [2 to 3 years](https://www.empowerit.com.au/blog/it-planning/budgeting/laptops-desktops-how-long-do-they-last/). While a laptop might survive longer, its utility will be limited as the components become less capable of running advanced applications. To determine the cadence for a laptop replacement, consider factors such as the laptops' age and condition, usage patterns, performance, budget, and financial goals, for example, if the laptop specs are high enough the good rule of thumb is every 5 years

Reviewer: ```suggestion
Most experts estimate a laptop’s lifespan to be [2 to 3 years](https://www.empowerit.com.au/blog/it-planning/budgeting/laptops-desktops-how-long-do-they-last/). While a laptop might survive longer than its lifespan, its utility will be limited as the components become less capable of running advanced applications. However, many factors influence this lifespan, such as the quality of specifications and usage patterns. If buying high-spec laptops, a good rule of thumb is every 5 years.
```

Author: Done - Updated the changes
